cl/corpus_importer/management/commands/scrape_pacer_free_opinions.py

```diff
@@ -299,9 +388,43 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
             default=False,
             help="Do we index as we go, or leave that to be done later?",
         )
+        parser.add_argument(
+            "--courts",
+            type=str,
+            default=["all"],
+            nargs="*",
+            help="The courts that you wish to parse. Use cl ids.",
+        )
+        parser.add_argument(
+            "--date-start",
+            dest="date_start",
+            required=False,
+            type=valid_date,
+            help="Date when the query should start.",
+        )
+        parser.add_argument(
+            "--date-end",
+            dest="date_end",
+            required=False,
+            type=valid_date,
+            help="Date when the query should end.",
+        )
 
     def handle(self, *args: List[str], **options: OptionsType) -> None:
         super().handle(*args, **options)
+
+        if options["date_start"] and not options["date_end"]:
+            logger.info(
+                "Error: date-end must be specified to use date-start option."
+            )
+            return
+
+        if options["date_start"] and options["date_end"]:
+            if options["date_start"] > options["date_end"]:  # type: ignore
+                logger.info(
+                    "Error: date-end must be greater or equal than date-start option."
+                )
+                return
```

Comment:
I think you are missing a scenario 

        date_start = options.get("date_start", None)
        date_end = options.get("date_end", None)

        if date_start:
            if not date_end or date_end < date_start:
                logger.error("Check dates: both dates must be present and correct")
                return
        elif date_end:
            logger.error("Must have start date")
            return

maybe this would work